token-stakedrop/stakedrop-merkle-generator.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+WORKDIR=$PWD
+
+# default file for calculated amounts of stakedrop for KEEP holders / stakers
+KEEP_AMOUNTS_BY_HOLDERS_PATH="$WORKDIR/output/result.json"
+
+NODE_CURRENT_VER="$(node --version)"
+NODE_REQUIRED_VER="v14.3.0"
+
+if [ "$(printf '%s\n' "$NODE_REQUIRED_VER" "$NODE_CURRENT_VER" | sort -V | head -n1)" != "$NODE_REQUIRED_VER" ]; 
+then
+      echo "Required node version must be at least ${NODE_REQUIRED_VER}" 
+      exit 1
+fi
+
+help()
+{
+   echo -e "\nUsage: ENV_VAR(S) $0"\
+           "--target-block <block_number>"
+   echo -e "\nEnvironment variables:\n"
+   echo -e "\tETH_ACCOUNT_PRIVATE_KEY: Ethereum account private key\n"\
+           "\tETH_HOSTNAME: Ethereum endpoint hostname"
+   echo -e "\nCommand line arguments:\n"
+   echo -e "\t--target-block: Block height when the stakedrop happens"
+   exit 1 # Exit script after printing help
+}
+
+# Transform long options to short ones
+for arg in "$@"; do
+  shift
+  case "$arg" in
+    "--target-block") set -- "$@" "-t" ;;
+    "--help")         set -- "$@" "-h" ;;
+    *)                set -- "$@" "$arg"
+  esac
+done
+
+# Parse short options
+OPTIND=1
+while getopts "t:h" opt
+do
+   case "$opt" in
+      t ) target_block="$OPTARG" ;;
+      h ) help ;;
+      ? ) help ;; # Print help in case parameter is non-existent
+   esac
+done
+shift $(expr $OPTIND - 1) # remove options from positional parameters
+
+# enter ETH_ACCOUNT_PRIVATE_KEY if not provided
+if [[ -z "$ETH_ACCOUNT_PRIVATE_KEY" ]]; then
+  read -p "Enter Ethereum account private key: " ETH_ACCOUNT_PRIVATE_KEY
+fi
```

Comment:
Why do we need ETH account private key? We are not writing anything to the chain.

Reply:
Ah, I think maybe it was used in [this PR](https://github.com/keep-network/keep-core/pull/2350) at some point but can't find any references now. I put it here because I thought it is used in our KEEP tracker scripts. 
I'll update

Reply:
5d92430